src/libs/README.md

```diff
@@ -1,3 +1,10 @@
 # Libs
-This section documents meta processes by the libs team.
 
+This section documents meta processes by the Libs team.
+
+## Where to find us
+
+The libs team hangs out primarily in [the rust-lang Zulip](https://rust-lang.zulipchat.com/) these days in the `#t-libs` stream.
+You can find more details about Zulip and how the Rust community uses it [here](../../chat/zulip).
+
+The `#rustc` IRC channel on `irc.mozilla.org` is in a "quasi-deprecated" status and is not recommended.
```

Comment:
The IRC channels are fully deprecated and no longer exist AFAIK, Mozilla has taken down their IRC servers so it might be best to remove or change this part.
```suggestion
```